test/bin/mdx-test/expect/casual-file-inc/test-case.opts

```diff
@@ -0,0 +1 @@
+--direction=to-md
```

Comment:
I think it makes sense for this test to remove this option as it is unrelated to its behaviour.

To clarify, this test has value whether the `--direction` option exists or not whereas the second one is here explicitly to show it's behaviour is independent from the option value, in particular when it is `to-ml`.
When we remove the option, we'll delete the second test but should be able to leave this one unchanged.

What do you think?

Reply:
Agreed, done in 32e4952